modules/manage/pages/audit-logging.adoc

```diff
@@ -0,0 +1,130 @@
+* EndTxn
+* InitProducerId
+* Produce
+
+|consume
+a|* AddOffsetsToTxn
+* Fetch
+* JoinGroup
+* LeaveGroup
+* ListOffset
+* OffsetCommit
+* SyncGroup
+* TxOffsetCommit
+
+|describe
+a|* DescribeACls
+* DescribeConfigs
+* DescribeGroups
+* DescribeLogDirs
+* FindCoordinator
+* ListGroups
+* ListPartitionReassginments
+* Metadata
+* OffsetForLeaderEpoch
+* DescribeProducers
+* DescribeTransations
+* ListTransactions
+
+|heartbeat
+a|* Heartbeat
+
+|authenticate
+a|* All authentication events
+
+|schema_registry
+a|* All Schema Registry API calls
+
+|admin
+a|* All Admin API calls
+|===
+
+
+== Enabling audit logging
+
+All audit log settings are applied at the cluster level using rpk. Use the `rpk cluster config` to configure audit logs. Some options will require a cluster restart. You can confirm this using `rpk cluster config status`.
+
+Some key steps to tuning your audit logging settings include:
+
+* If you wish to change the number of partitions or the replication factor for your audit log topic, set the `audit_log_num_partitions` and `audit_log_replication_factor` properties respectively.
+* Choose the type of events needed by setting `audit_enabled_event_types` to the desired list of event categories. Keep this as restrictive as possible based on your compliance and security needs to avoid excessive noise in your audit logs.
+* Identify non-sensitive topics so that you can exclude them from auditing. Set `audit_exclude_topics` to this list of topics.
+* Identify non-sensitive principles so that you can exclude them from auditing. Set `audit_exclude_principles` to this list of principles. This command accepts names in the form of `name` or `User:name`.
+* Set `audit_enabled` to `true`.
+* Assess the retention needs for your audit logs. You may not need to keep the logs around for the default seven days. This is controlled by setting `retention.ms` for the `__audit_log` topic or by setting `delete_retention_ms` at the cluster level.
+
+The sequence of commands in rpk for these steps is as follows:
+
+ rpk cluster config set audit_log_num_partitions 6
+ rpk cluster config set audit_log_replication_factor 5
+ rpk cluster config set audot_enabled_event_types '["management","describe","authenticate"]'
+ rpk cluster config set audit_exclude_topics '["topic1","topic2"]'
+ rpk cluster config set audit_exclude_principles '["User:principle1", "principle2"]'
+ rpk cluster config set audit_enabled true
+ rpk topic alter-config __audit_log --set retention.ms=259200000
+
+== Optimizing costs for audit logging
+
+By default, audit logging is turned off. When enabled, audit logging can quickly generate a very large amount of data, especially if all event types are selected. Proper configuration of audit logging is critical for avoiding filling your disk or using excess tiered storage. The configuration options available help ensure your audit logs contain only the volume of data necessary to meeting your regulatory or legal requirements.
```

Comment:
```suggestion
By default, audit logging is disabled. When enabled, audit logging can quickly generate a very large amount of data, especially if all event types are selected. Proper configuration of audit logging is critical to avoid filling your disk or using excess Tiered Storage. The configuration options available help ensure your audit logs contain only the volume of data necessary to meeting your regulatory or legal requirements.
```